website/static/css/custom.css

```diff
@@ -72,6 +72,11 @@ article a:hover {
   color: var(--base-walnut-color);
 }
 
+article p img {
+  margin-left: 0;
+  display: inline-block;
+}
+
```

Comment:
This is really unspecific, don't badges have some kind of selectors on them?

Reply:
There are no options to select specific images.
This won't break the layout of the image, it will just push it to the left.